### PR TITLE
Ntbs 2504

### PR DESCRIPTION
**Pull request by aptemus**
* a new view which will create the NOIDS extract for both ETS and NTBS records
* a new table which will keep a count of the records which existed at a given point in time (i.e. each Monday when the NOIDS file is sent, count of all the records that exist at that point with a notification date on or before the Sunday before). Currently Rosanne maintains this count manually in a Word document

I do have some outstanding questions for her about what week number is used to record the counts, but this can be resolved in Power BI; as there are different ways of calculating week numbers, I have opted to record the count against a date, so the week number can be calculated later without affecting the underlying data.

### CODE REVIEW

File: source/dbo/Stored Procedures/Power BI Reporting/uspNOIDSWeeklyCount.sql [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+CREATE PROCEDURE [dbo].[uspGenerateNOIDSWeeklyCount]
+AS
+	DECLARE @ThisWeek VARCHAR(10) =  (SELECT ISOYearWeek FROM [dbo].Calendar WHERE DateValue = CONVERT(DATE, GETUTCDATE()))
+	DECLARE @LastSunday DATE =  (SELECT MAX(DateValue) FROM [dbo].Calendar WHERE DateValue <= GETUTCDATE() AND DATENAME(dw, DateValue) = 'Sunday')
+
+	--check to see if a value has already been stored for this week
+	IF NOT EXISTS
+	(SELECT wc.DateValue 
+		FROM [dbo].[NOIDSWeeklyCount] wc
+			INNER JOIN [dbo].[Calendar] c ON c.DateValue = wc.DateValue
```
Comment (JoshCadney) [inline source/dbo/Stored Procedures/Power BI Reporting/uspNOIDSWeeklyCount.sql:12]: This join is confusing me... it looks like `DateValue` in `Calendar` is just a date, whereas `DateValue` in `NOIDSWeeklyCount` is the exact time of a previous run, so this won't ever match anything?

**Review by adilmirzaphe — COMMENTED** [review]
Added Lookup for LocalAuthority and updated code

**Review by benjimarshall — APPROVED** [review]
Looks good!

**Review by adilmirzaphe — APPROVED** [review]
looks good

**Review by JoshCadney — APPROVED** [review]
Looks good :-)

**Pull request closed without merging**